ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java

```diff
@@ -399,6 +400,7 @@ public Eth2NetworkConfiguration build() {
             SpecFactory.create(
                 constants,
                 builder -> {
+                  EphemeryNetwork.updateConfig(builder);
```

Comment:
this will be applying these defaults to all the networks, so that'll explain why the AT is breaking... Just as an observation, I haven't proven it, but this will apply to all networks here.

Reply:
Making update now, thanks